src/components/DeleteMember/DeleteMemberMutation.js

```diff
@@ -0,0 +1,14 @@
+import { gql } from 'react-apollo';
+
+export default gql`
+  mutation DeleteMemberMutation($id: Int!) {
+    deleteMember(id: $id) {
+      discordName
+      email
+      id
+      membershipType
+      admin
+      currentUserStatus
```

Comment:
Is the convention for delete on GraphQL to return the deleted object? Not sure what the best practices for this are.

Reply:
Often times in mutations you would want to return the object. But in this case I don't think it would be necessary since I'm not using what returned to update the state or the store.

Reply:
True, and usually with Apollo you don't update the store directly, you would trigger a refetch query instead.